deploy_vpipe.sh

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env bash
```

Comment:
Since the recommended way of executing this installer involves downloading the script via HTTP (presumably via a hyperlink provided in a tutorial), I see no need to have this script in the project root directory.

Although it can be convenient/helpful for novices, this script is not going to be the preferred mode of installation for advanced users who exercise control over their conda environments and other software.

Therefore, please rename the script to `utils/quick_install.sh`.

Reply:
Advanced users: I literally say that at the begin of the tutorial (see PR 39).